exercises/basics/types.ts

```diff
@@ -1,3 +1,15 @@
 export function returnAllTypesObject() : {[key : string] : any} {
   // @TODO implement
```

Comment:
@vuhungthang  remove this line

anytime you actually do the real implementation, you don't need to keep the `@TODO` comment anymore